ios/FluentUI/Navigation/Views/AvatarTitleView.swift

```diff
@@ -76,7 +76,13 @@ class AvatarTitleView: UIView, TokenizedControlInternal, TwoLineTitleViewDelegat
         didSet {
             updateAppearance()
             twoLineTitleView.currentStyle = style == .primary ? .primary : .system
-            avatar?.state.style = style == .primary ? .default : .accent
+            let avatarStyle: MSFAvatarStyle
+            if let overrideStyle = avatarOverrideStyle {
```

Comment:
Nit: we're moving towards simpler unwrapping in the repo. This line could just be `if let avatarOverrideStyle`